lib/oli_web/views/delivery_view.ex

```diff
@@ -2,11 +2,14 @@ defmodule OliWeb.DeliveryView do
   use OliWeb, :view
   use Phoenix.Component
 
+  import OliWeb.Common.SourceImage
```

Comment:
Is this import necessary?

Reply:
I assumed it is because we are using the functions defined there in some delivery templates, right? I wonder why we add the `Routes` alias below?

Reply:
Discussed offline - needed by the enroll template.

Reply:
Routes alias not needed, removing